exporters/logging/src/main/java/io/opentelemetry/exporters/logging/LoggingMetricExporter.java

```diff
@@ -41,16 +42,16 @@ public ResultCode export(Collection<MetricData> metrics) {
    * @return the result of the operation
    */
   @Override
-  public ResultCode flush() {
-    ResultCode resultCode = ResultCode.SUCCESS;
+  public CompletableResultCode flush() {
+    CompletableResultCode resultCode = new CompletableResultCode();
     for (Handler handler : logger.getHandlers()) {
       try {
         handler.flush();
       } catch (Throwable t) {
-        resultCode = ResultCode.FAILURE;
+        resultCode.fail();
```

Comment:
Should we early return here? Similar comment elsewhere

Reply:
I think it doesn't matter in this particular case, since we might want to flush all the handlers, even if one fails. The call to succeed() below is a no-op if it's been previously failed.

Reply:
I'll add a return to make it clear but, as John mentioned, it makes no diff.